docs/kyma/05-05-profiles.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Profiles
+type: Configuration
+---
+
+The Kyma Operator allows you to use pre-defined profiles to install or upgrade Kyma. To install Kyma using a profile, you must specify the **spec.profile** attribute in the [Installation custom resource](#custom-resource-installation) (CR).
```

Comment:
I'd add general info about profiles in the 2nd sentence and move `To install Kyma using a profile...` part to the next paragraph (or even after the paragraph explaining values overrides, we'll see 🤔 ).